exporter/newrelicexporter/newrelic.go

```diff
@@ -64,8 +64,8 @@ type exporter struct {
 	logger             *zap.Logger
 }
 
-func clientOptions(apiKey string, apiKeyHeader string, hostOverride string, insecure bool) []telemetry.ClientOption {
-	options := []telemetry.ClientOption{telemetry.WithUserAgent(product + "/" + version)}
+func clientOptions(info component.ApplicationStartInfo, apiKey string, apiKeyHeader string, hostOverride string, insecure bool) []telemetry.ClientOption {
```

Comment:
We can use a pointer for the new parameter.

Reply:
Should we accept just the version string since that's all that needed?

Reply:
We also use the executable name 😄